README.md

```diff
@@ -13,8 +13,7 @@ git clone git@github.com:ObolNetwork/charon-docker-compose.git
 cd charon-docker-compose
 make                                   # Shows available make targets
 make clean                             # Deletes previously created cluster
-make create-cluster                    # Creates simnet cluster
-docker-compose up
+make up
```

Comment:
need `make split-existing-keys` section before this

Reply:
also need one more step to copy .env.sample to .env and fill in the required variables

Reply:
![Screenshot 2022-05-17 at 12 14 52](https://user-images.githubusercontent.com/4981644/168798647-8987f650-b8ca-4df6-8674-a426b3229be1.png)

Agreed, have been building an obol-docs page alongside this with this info. Will take the time to duplicate the copy here when its all finished.